scrum/policies.md

```diff
@@ -206,6 +206,7 @@ There are various formats that a card's Story Points will go under after the com
 
 - `1SP (was 1SP, ...)` - 1SP planned this sprint, 1 left over from last sprint.
 - `2SP + 1 retro` - Additional 2 story points required and updated after release.
```

Comment:
Shouldn't this be Additional 1 SP required after release, etc?

Reply:
I am not sure what you are asking here?

Reply:
Discussed w/ Duncan over zoom, resolved